src/users.js

```diff
@@ -0,0 +1,47 @@
+export const setCurrentUser = (customers, id) => {
+  return customers.find(customer => customer.id === id);
+}
+
+export const filterRoomsByDate = (selectDate, bookings, rooms) => {
+  const searchResult = rooms.reduce((availRooms, room) => {
+    const match = bookings.find(({ date, roomNumber }) => {
+      return selectDate !== date && room.number === roomNumber
+    })
+    if (match) availRooms.push(room);
+    return availRooms;
+  }, [])
+  if (searchResult.length === 0) {
```

Comment:
Totally up to your preference, but you might consider putting this logic in your DOM updates and leaving this function to return data only (i.e. if searchResult is an empty array, just return the empty array and then in the DOM use a conditional that 'if the array is empty' target an element.innerText to insert your apology line). It might be a cleaner separation between your data model logic and your DOM logic.